### DISCUSSION TITLE
Will there be an Android app?

### DISCUSSION THREAD

**Discussion by ghost** [toeverything/AFFiNE#4325] [Q&A] [answered]
Hi, I'd really like to switch from Notion to AFFiNE but the one thing preventing me is that there isn't an Android app currently. Are you planning on developing a mobile app? Thanks!

**Comment by houjoe0829** [comment, marked as answer]
Thank you very much for your attention. Mobile support will be a very important milestone for AFFiNE and is already on our roadmap. Please continue to follow our updates.
